perun-core/src/main/java/cz/metacentrum/perun/core/implApi/modules/attributes/UserVirtualAttributeCollectedFromUserExtSource.java

```diff
@@ -158,6 +158,7 @@ public Attribute getAttributeValue(PerunSessionImpl sess, User user, AttributeDe
 	@Override
 	public List<String> resolveVirtualAttributeValueChange(PerunSessionImpl perunSession, String message) throws InternalErrorException, WrongReferenceAttributeValueException, AttributeNotExistsException, WrongAttributeAssignmentException {
 		List<String> resolvingMessages = new ArrayList<>();
+		log.debug("Received message {}", message);
```

Comment:
This log is too frequent. Please remove it and add only "info" log to the Auditer class itself where we will have better control on using these debug messages.